lib/crowbar/client/app/backup.rb

```diff
@@ -54,13 +54,13 @@ class Backup < Base
           type: :boolean,
           default: false,
           aliases: [],
-          desc: "Format output as table, a shortcut for --format table option"
+          desc: "Format output as json, a shortcut for --format json option"
 
         method_option :plain,
           type: :boolean,
           default: false,
           aliases: [],
-          desc: "Format output as table, a shortcut for --format table option"
+          desc: "Format output as plain, a shortcut for --format plain option"
```

Comment:
What about _as plain text_?

Reply:
+1, or only `text`

Reply:
Thanks @jsuchome and @aplanas ! Will do.